saleor/graphql/webhook/tests/mutations/test_webhook_dry_run.py

```diff
@@ -196,441 +193,6 @@ def test_webhook_dry_run_event_type_not_supported(
-            subscription_permission_group_created_webhook,
-            permission_group_manage_users,
-        ],
-        events.PERMISSION_GROUP_UPDATED: [
-            subscription_permission_group_updated_webhook,
-            permission_group_manage_users,
-        ],
-        events.PERMISSION_GROUP_DELETED: [
-            subscription_permission_group_deleted_webhook,
-            permission_group_manage_users,
-        ],
-        events.SHIPPING_PRICE_CREATED: [
-            subscription_shipping_price_created_webhook,
-            shipping_method,
-        ],
-        events.SHIPPING_PRICE_UPDATED: [
-            subscription_shipping_price_updated_webhook,
-            shipping_method,
-        ],
-        events.SHIPPING_PRICE_DELETED: [
-            subscription_shipping_price_deleted_webhook,
-            shipping_method,
-        ],
-        events.SHIPPING_ZONE_CREATED: [
-            subscription_shipping_zone_created_webhook,
-            shipping_zone,
-        ],
-        events.SHIPPING_ZONE_UPDATED: [
-            subscription_shipping_zone_updated_webhook,
-            shipping_zone,
-        ],
-        events.SHIPPING_ZONE_DELETED: [
-            subscription_shipping_zone_deleted_webhook,
-            shipping_zone,
-        ],
-        events.SHIPPING_ZONE_METADATA_UPDATED: [
-            subscription_shipping_zone_metadata_updated_webhook,
-            shipping_zone,
-        ],
-        events.STAFF_CREATED: [subscription_staff_created_webhook, staff_user],
-        events.STAFF_UPDATED: [subscription_staff_updated_webhook, staff_user],
-        events.STAFF_DELETED: [subscription_staff_deleted_webhook, staff_user],
-        events.TRANSACTION_ITEM_METADATA_UPDATED: [
-            subscription_transaction_item_metadata_updated_webhook,
-            transaction_item_created_by_app,
-        ],
-        events.TRANSLATION_CREATED: [
-            subscription_translation_created_webhook,
-            translated_attribute,
-        ],
-        events.TRANSLATION_UPDATED: [
-            subscription_translation_updated_webhook,
-            translated_attribute,
-        ],
-        events.VOUCHER_CREATED: [subscription_voucher_created_webhook, voucher],
-        events.VOUCHER_UPDATED: [subscription_voucher_updated_webhook, voucher],
-        events.VOUCHER_DELETED: [subscription_voucher_deleted_webhook, voucher],
-        events.VOUCHER_METADATA_UPDATED: [
-            subscription_voucher_metadata_updated_webhook,
-            voucher,
-        ],
-        events.WAREHOUSE_CREATED: [subscription_warehouse_created_webhook, warehouse],
-        events.WAREHOUSE_UPDATED: [subscription_warehouse_updated_webhook, warehouse],
-        events.WAREHOUSE_DELETED: [subscription_warehouse_deleted_webhook, warehouse],
-        events.WAREHOUSE_METADATA_UPDATED: [
-            subscription_warehouse_metadata_updated_webhook,
-            warehouse,
-        ],
-    }
-
-
-def test_webhook_dry_run_root_type(
```

Comment:
Why do we delete this?

Reply:
this one should remain, It is good you noticed it